### DISCUSSION TITLE
Diff update 4.3.1 - 4.3.1.1

### DISCUSSION THREAD

**Discussion by 7zxkv** [OnionUI/Onion#1617] [General]
@Aemiii91 

it's possible that you could give a simple patch between 4.3.1 and 4.3.1.1 because the difference is almost nothing, so it'd be nice to avoid having to do it all over again for so little, thanks :D

**Comment by Aemiii91** [comment]
Here you go: [Onion_patch_v4.3.1_to_v4.3.1-1.zip](https://github.com/user-attachments/files/15973227/Onion_patch_v4.3.1_to_v4.3.1-1.zip)

**Reply by 7zxkv** [reply]
thanks you !! :D
